src/ui/components/TabHeader.jsx

```diff
@@ -22,10 +22,12 @@ const TabHeader = props => (
 
 TabHeader.propTypes = {
   selectedIndex: React.PropTypes.number,
-  members: React.PropTypes.arrayOf({
-    id: React.PropTypes.string,
-    name: React.PropTypes.string,
-  }).isRequired,
+  members: React.PropTypes.arrayOf(
```

Comment:
Ich binde noch "eslint" in meinen Editor ein. Dann ist es auch sauber von meiner Seite.